app/models/user.rb

```diff
@@ -371,10 +369,9 @@ def name_and_email
   # Returns list of requests which the user hasn't described (and last
   # changed more than a day ago)
   def get_undescribed_requests
-    info_requests.where(
-      "awaiting_description = ? and #{ InfoRequest.last_event_time_clause } < ?",
-      true, 1.day.ago
-    )
+    info_requests.
+      where(awaiting_description: true).
+      where("#{ InfoRequest.last_event_time_clause } < ?", 1.day.ago)
```

Comment:
```suggestion
      where(InfoRequest.last_event_time_clause => (...1.day.ago))
```

Reply:
Fancy! I'm not sure I fully understand what this does, so I'm going to merge this PR as is. Plz do make a quick PR with this so that you can school me and also get the credit for it 😄